cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -56,8 +56,10 @@ spec:
       interval: 60s
   # quota in bytes and/or objects, default value is 0 (unlimited)
   # see https://docs.ceph.com/en/latest/rados/operations/pools/#set-pool-quotas
+  # do not set both maxBytes and maxBytesHuman, choose one or the other
```

Comment:
Docs and examples would only need to mention `maxSize`, per other comment

Reply:
changes submitted, thx